LRQA-57134 | master

xbrianlee opened the pull request:
https://issues.liferay.com/browse/LRQA-57134

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

xbrianlee commented:
ci:test:sf

xbrianlee commented:
ci:test:relevant

xbrianlee commented:
ci:test:security

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-3.liferay.com/job/test-portal-source-format/3485/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-3.liferay.com//userContent/jobs/test-portal-source-format/builds/3485/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest(master)/3270/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-18.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/3270/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 5 minutes 41 seconds 180 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/c3fe9f2af73a694df4351498ce242f0da18bbe33">c3fe9f2af73a694df4351498ce242f0da18bbe33</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/xbrianlee/liferay-portal/tree/LRQA-57134">LRQA-57134</a><br/>Branch GIT ID: <a href="https://github.com/xbrianlee/liferay-portal/commit/d51574298b2182d7badee6ec26338b4c802d43da">d51574298b2182d7badee6ec26338b4c802d43da</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-3.liferay.com/job/test-portal-source-format/3485/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-3.liferay.com/userContent/jobs/test-portal-source-format/builds/3485/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest(master)/6780/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-11.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/6780/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 13 out of 13 jobs passed</h3><h3>:heavy_check_mark: ci:test:relevant - 19 out of 19 jobs passed in 1 hour 20 minutes 9 seconds 999 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/4cf384e0e7e1045aeb93b03a9330527cd7c0786b">4cf384e0e7e1045aeb93b03a9330527cd7c0786b</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/b4536024712dac54820c4c13f7cf737b2b7a945b">b4536024712dac54820c4c13f7cf737b2b7a945b</a></p><details><summary><strong>ci:test:stable - 13 out of 13 jobs PASSED</strong></summary><details><summary><strong>13 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/258139/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/347475/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/386132/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/1</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/402947/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/10</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/381825/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/11</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/381808/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/2</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/381824/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/3</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/394918/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/4</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/386898/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/5</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/386133/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/6</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/251991/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/7</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/381809/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/8</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/394919/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/9</a></li></ul></details></details><details><summary><strong>ci:test:relevant - 19 out of 19 jobs PASSED</strong></summary><details><summary><strong>19 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest(master)/6780/">test-portal-acceptance-pullrequest(master)</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/276692/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/258139/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/378214/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/386899/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/358475/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/378213/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/347475/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/386132/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/1</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/402947/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/10</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/381825/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/11</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/381808/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/2</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/381824/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/3</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/394918/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/4</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/386898/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/5</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/386133/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/6</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/251991/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/7</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/381809/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/8</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/394919/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/9</a></li></ul></details></details><h5>For more details click <a href="https://test-1-11.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/6780/jenkins-report.html">here</a>.</h5></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://files.liferay.com/private/bundles/test-1-11/jobs/test-portal-acceptance-pullrequest(master)/builds/6780/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a> (mirrors: <a href="http://mirrors.dlc.liferay.com/files.liferay.com/private/bundles/test-1-11/jobs/test-portal-acceptance-pullrequest(master)/builds/6780/liferay-portal-bundle-tomcat.tar.gz">dlc</a>, <a href="http://mirrors.lax.liferay.com/files.liferay.com/private/bundles/test-1-11/jobs/test-portal-acceptance-pullrequest(master)/builds/6780/liferay-portal-bundle-tomcat.tar.gz">lax</a>)</li></ul>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:security - 47 out of 56 jobs passed in 1 hour 45 minutes 7 seconds 408 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/4cf384e0e7e1045aeb93b03a9330527cd7c0786b">4cf384e0e7e1045aeb93b03a9330527cd7c0786b</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/b4536024712dac54820c4c13f7cf737b2b7a945b">b4536024712dac54820c4c13f7cf737b2b7a945b</a></p><details><summary><strong>ci:test:security - 47 out of 56 jobs PASSED</strong></summary><div><h4>9 Failed Jobs:</h4><ul><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest(master)/3270/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/348301/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/20</strong></strike></a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/326065/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/26</strong></strike></a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/311800/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/28</strong></strike></a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/386129/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/3</strong></strike></a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/381805/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/32</strong></strike></a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/378211/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/37</strong></strike></a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/394914/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/4</strong></strike></a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/348303/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/47</strong></strike></a></li></ul></div><details><summary><strong>47 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/347470/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/261214/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/326067/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/378210/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/10</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/311801/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/11</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/386894/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/12</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/376860/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/13</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/348302/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/14</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/386893/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/15</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/381802/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/16</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/390748/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/17</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/381803/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/18</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/313909/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/19</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/358472/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/381801/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/21</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/386128/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/22</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/394913/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/23</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/348300/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/24</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/386127/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/25</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/326064/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/27</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/347467/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/29</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/386896/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/30</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/313910/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/31</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/326069/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/33</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/347471/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/34</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/311803/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/35</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/251987/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/36</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/390749/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/38</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/348304/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/39</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/326068/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/40</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/376861/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/41</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/386895/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/42</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/381804/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/43</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/394915/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/44</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/381821/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/45</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/276691/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/46</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/311802/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/5</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/261213/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/6</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/390747/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/7</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/326066/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/8</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/347469/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/9</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/358473/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/376862/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/261215/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/375255/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/386130/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/347468/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8</a></li></ul></details></details><h5>For more details click <a href="https://test-1-18.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/3270/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/348301/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/20</a></h5><div><h6>Job Results:</h6><p>1 Test Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/348301//consoleText">AXIS_VARIABLE=0,label_exp=!master #348301</a><ol><li><div><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/348301//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_OAuth2_AuthorizeExternalApplication_">LocalFile.OAuth2#AuthorizeExternalApplication</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-18/1587399751179/test-portal-acceptance-pullrequest(master)/3270/functional-tomcat90-mysql57-jdk8/20/0/LocalFile.OAuth2_AuthorizeExternalApplication/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-18/1587399751179/test-portal-acceptance-pullrequest(master)/3270/functional-tomcat90-mysql57-jdk8/20/0/LocalFile.OAuth2_AuthorizeExternalApplication/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-18/1587399751179/test-portal-acceptance-pullrequest(master)/3270/functional-tomcat90-mysql57-jdk8/20/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/326065/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/26</a></h5><div><h6>Job Results:</h6><p>1 Test Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/326065//consoleText">AXIS_VARIABLE=0,label_exp=!master #326065</a><ol><li><div><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/326065//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_Login_OpenIDConnectSmoke_">LocalFile.Login#OpenIDConnectSmoke</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-18/1587399751179/test-portal-acceptance-pullrequest(master)/3270/functional-tomcat90-mysql57-jdk8/26/0/LocalFile.Login_OpenIDConnectSmoke/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-18/1587399751179/test-portal-acceptance-pullrequest(master)/3270/functional-tomcat90-mysql57-jdk8/26/0/LocalFile.Login_OpenIDConnectSmoke/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-18/1587399751179/test-portal-acceptance-pullrequest(master)/3270/functional-tomcat90-mysql57-jdk8/26/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/311800/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/28</a></h5><div><h6>Job Results:</h6><p>1 Test Passed.<br/>5 Tests Failed.</p></div><ol><li><div><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/311800//consoleText">AXIS_VARIABLE=0,label_exp=!master #311800</a><ol><li><div><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/311800//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_XSS_ValidateXSSVulnerabilities_">LocalFile.XSS#ValidateXSSVulnerabilities</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-18/1587399751179/test-portal-acceptance-pullrequest(master)/3270/functional-tomcat90-mysql57-jdk8/28/0/LocalFile.XSS_ValidateXSSVulnerabilities/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-18/1587399751179/test-portal-acceptance-pullrequest(master)/3270/functional-tomcat90-mysql57-jdk8/28/0/LocalFile.XSS_ValidateXSSVulnerabilities/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-18/1587399751179/test-portal-acceptance-pullrequest(master)/3270/functional-tomcat90-mysql57-jdk8/28/0/jenkins-console.txt.gz">Console Output</a></div></li><li><div><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/311800//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_XSS_ViewLiferayPortletList_">LocalFile.XSS#ViewLiferayPortletList</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-18/1587399751179/test-portal-acceptance-pullrequest(master)/3270/functional-tomcat90-mysql57-jdk8/28/0/LocalFile.XSS_ViewLiferayPortletList/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-18/1587399751179/test-portal-acceptance-pullrequest(master)/3270/functional-tomcat90-mysql57-jdk8/28/0/LocalFile.XSS_ViewLiferayPortletList/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-18/1587399751179/test-portal-acceptance-pullrequest(master)/3270/functional-tomcat90-mysql57-jdk8/28/0/jenkins-console.txt.gz">Console Output</a></div></li><li><div><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/311800//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_XSS_ViewLookAndFeel_">LocalFile.XSS#ViewLookAndFeel</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-18/1587399751179/test-portal-acceptance-pullrequest(master)/3270/functional-tomcat90-mysql57-jdk8/28/0/LocalFile.XSS_ViewLookAndFeel/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-18/1587399751179/test-portal-acceptance-pullrequest(master)/3270/functional-tomcat90-mysql57-jdk8/28/0/LocalFile.XSS_ViewLookAndFeel/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-18/1587399751179/test-portal-acceptance-pullrequest(master)/3270/functional-tomcat90-mysql57-jdk8/28/0/jenkins-console.txt.gz">Console Output</a></div></li><li>...</li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/386129/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/3</a></h5><div><h6>Job Results:</h6><p>2 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/386129//consoleText">AXIS_VARIABLE=0,label_exp=!master #386129</a><ol><li><div><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/386129//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_OAuth2_AssertExpandPanelsOnGlobalScopes_">LocalFile.OAuth2#AssertExpandPanelsOnGlobalScopes</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-18/1587399751179/test-portal-acceptance-pullrequest(master)/3270/functional-tomcat90-mysql57-jdk8/3/0/LocalFile.OAuth2_AssertExpandPanelsOnGlobalScopes/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-18/1587399751179/test-portal-acceptance-pullrequest(master)/3270/functional-tomcat90-mysql57-jdk8/3/0/LocalFile.OAuth2_AssertExpandPanelsOnGlobalScopes/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-18/1587399751179/test-portal-acceptance-pullrequest(master)/3270/functional-tomcat90-mysql57-jdk8/3/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/381805/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/32</a></h5><div><h6>Job Results:</h6><p>4 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/381805//consoleText">AXIS_VARIABLE=0,label_exp=!master #381805</a><ol><li><div><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/381805//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_XSS_ViewMobileDeviceFamilies_">LocalFile.XSS#ViewMobileDeviceFamilies</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-18/1587399751179/test-portal-acceptance-pullrequest(master)/3270/functional-tomcat90-mysql57-jdk8/32/0/LocalFile.XSS_ViewMobileDeviceFamilies/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-18/1587399751179/test-portal-acceptance-pullrequest(master)/3270/functional-tomcat90-mysql57-jdk8/32/0/LocalFile.XSS_ViewMobileDeviceFamilies/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-18/1587399751179/test-portal-acceptance-pullrequest(master)/3270/functional-tomcat90-mysql57-jdk8/32/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li>...</li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream(master)">here</a>.</h4></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://files.liferay.com/private/bundles/test-1-18/jobs/test-portal-acceptance-pullrequest(master)/builds/3270/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a> (mirrors: <a href="http://mirrors.dlc.liferay.com/files.liferay.com/private/bundles/test-1-18/jobs/test-portal-acceptance-pullrequest(master)/builds/3270/liferay-portal-bundle-tomcat.tar.gz">dlc</a>, <a href="http://mirrors.lax.liferay.com/files.liferay.com/private/bundles/test-1-18/jobs/test-portal-acceptance-pullrequest(master)/builds/3270/liferay-portal-bundle-tomcat.tar.gz">lax</a>)</li></ul>

xbrianlee commented:
ci:forward

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

The pull request will automatically be forwarded to the user `brianchandotcom` if the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-16.liferay.com/job/test-portal-source-format/3232/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-16.liferay.com//userContent/jobs/test-portal-source-format/builds/3232/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 3 minutes 20 seconds 191 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/640344e337a134111d6bc02ee61a78a375de2665">640344e337a134111d6bc02ee61a78a375de2665</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/xbrianlee/liferay-portal/tree/LRQA-57134">LRQA-57134</a><br/>Branch GIT ID: <a href="https://github.com/xbrianlee/liferay-portal/commit/b3c9f67313a710f06285ae726430fca331ef236b">b3c9f67313a710f06285ae726430fca331ef236b</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-16.liferay.com/job/test-portal-source-format/3232/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-16.liferay.com/userContent/jobs/test-portal-source-format/builds/3232/jenkins-report.html">here</a>.</h5></details></html>

xbrianlee commented:
@gaborlovasdotcom Nice job! A couple of fixes below.
90d50dc: Removed two steps since they duplicate `OAuth2#AuthorizeExternalApplication`
b3c9f67: We should always define vars right before we use them. We didn't do a good job doing this for `OAuth2#AuthorizeExternalApplication` so I fixed it there as well.

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest(master)/5941/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-16.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5941/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 13 out of 13 jobs passed</h3><h3>:heavy_check_mark: ci:test:relevant - 19 out of 19 jobs passed in 2 hours 24 minutes 44 seconds 500 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/ec33dae1f67ad941b2fc025ccc201a5ce6a2b074">ec33dae1f67ad941b2fc025ccc201a5ce6a2b074</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/b4536024712dac54820c4c13f7cf737b2b7a945b">b4536024712dac54820c4c13f7cf737b2b7a945b</a></p><details><summary><strong>ci:test:stable - 13 out of 13 jobs PASSED</strong></summary><details><summary><strong>13 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/375298/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/381890/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/375299/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/1</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/376899/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/10</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/311851/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/11</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/381889/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/2</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/358509/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/3</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/381888/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/4</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/347532/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/5</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/376898/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/6</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/375301/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/7</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/381891/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/8</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/252081/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/9</a></li></ul></details></details><details><summary><strong>ci:test:relevant - 19 out of 19 jobs PASSED</strong></summary><details><summary><strong>19 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest(master)/5941/">test-portal-acceptance-pullrequest(master)</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/386174/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/375298/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/375300/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/358510/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/402988/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/386173/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/381890/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/375299/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/1</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/376899/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/10</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/311851/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/11</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/381889/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/2</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/358509/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/3</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/381888/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/4</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/347532/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/5</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/376898/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/6</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/375301/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/7</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/381891/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/8</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/252081/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/9</a></li></ul></details></details><h5>For more details click <a href="https://test-1-16.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5941/jenkins-report.html">here</a>.</h5></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://files.liferay.com/private/bundles/test-1-16/jobs/test-portal-acceptance-pullrequest(master)/builds/5941/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a> (mirrors: <a href="http://mirrors.dlc.liferay.com/files.liferay.com/private/bundles/test-1-16/jobs/test-portal-acceptance-pullrequest(master)/builds/5941/liferay-portal-bundle-tomcat.tar.gz">dlc</a>, <a href="http://mirrors.lax.liferay.com/files.liferay.com/private/bundles/test-1-16/jobs/test-portal-acceptance-pullrequest(master)/builds/5941/liferay-portal-bundle-tomcat.tar.gz">lax</a>)</li></ul>

liferay-continuous-integration commented:
All required test suite(s) passed.
Forwarding pullrequest to `brianchandotcom`.

liferay-continuous-integration commented:
Pull request has been successfully forwarded to https://github.com/brianchandotcom/liferay-portal/pull/87793
